src/gui/sdl_mapper.cpp

```diff
@@ -2767,7 +2767,7 @@ static bool load_binds_from_file(const std::string_view mapperfile_path,
 	// default, the mapperfile is not provided
 	if (!was_loaded && mapperfile_name != MAPPERFILE)
 		LOG_WARNING("MAPPER: Failed loading mapperfile '%s' directly or from resources",
-		            mapperfile_name.data());
+		            std::string(mapperfile_name).c_str());
```

Comment:
Can you check to see if the function signatures for some of these can go from `std::string_view` to `const std::string &`?  I'm not seeing the benefit of string view if you have to do this ugly thing to get a proper null terminated string out of it.

Reply:
I'll take a look at that. It won't catch all places, but where appropriate I think it's the better method.